managed-os/build.f/particles-managed-os/particles/conditions/isNeedToSkipSigAuthForDataDog.hbs

```diff
@@ -0,0 +1 @@
+"Fn::Or": [{"Fn::Equals":[{"Ref": "OSName"},"Windows Server 2008 SP2 64-bit"]},{"Condition": "is32BitOs"}]
```

Comment:
what is "is32BitOs"?

Reply:
How are we handling Win2k8 32-bit and 64-bit?

Reply:
I think is32BitOs condition check it.

Reply:
@RajkumarShivage  I am using "or" condition for Win2008 64bit or is32BitOs check. I have referred -  http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-conditions.html#intrinsic-function-reference-conditions-or for putting this condition

Reply:
@RajkumarShivage  **is32BitOs** condition checks its Win2008 32bit os. (Its previously added condition, I am just reusing along with new condition ie. win 2008 64bit check)